benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -451,6 +451,28 @@ msgstr ""
 msgid "You selected a Veteran transit benefit."
 msgstr ""
 
+msgid "Your card information may not have been entered correctly."
+msgstr ""
+
+#, python-brace-format
+msgid ""
+"The number and last name must be entered exactly as they appear on your "
+"{agency_card}. Please check your card and try again, or contact your transit "
```

Comment:
This is so cool! It would be great to go back and utilize this pattern on some of our other copy :rocket:

Reply:
Yes! This is the equivalent to `{% blocktranslate %}` in templates, figuring out how to do it lazily in Python was the key but it totally works!